README.md

```diff
@@ -94,32 +97,33 @@ user@host:~$ substrait-validator input.yaml --out-file output.html --mode ignore
 ```
 
 This generates `output.html`, a self-contained HTML file describing the plan.
-
 Just like the input file, the output file format is derived from the file
-extension, so the `.html` part is significant. If you don't want to rely on
-this, you can also just specify the formats you want manually using `--in-type`
-and `--out-type`.
+extension, so the `.html` part is significant.
 
-`--mode ignore` tells the validator to emit a file and exit with code 0
-regardless of the validation result. The full list of modes is:
+The validator only emits an output file if it exits with code 0, and exits
+with code 1 if validation fails, in order to play nicely with build systems
```

Comment:
```suggestion
with code 1 if validation fails, to play nicely with build systems
```

Reply:
I personally like the explicitness of "in order to" for signalling intent more than just "to", but people tend to find my writing too verbose so I'll accept this one.